src/helpers/RemoveEmptyFolders.ts

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
You are on fire =)

We can create another stateManagement component called rules where to incorporate this logic. So you can access to view component or initilialize whathever you want to use for

seeing that you are contemplating the cases of the incident that we commented on (https://github.com/RafaelGB/obsidian-db-folder/issues/183), it is an opportunity to focus it towards those future folder rules